src/github-utils.ts

```diff
@@ -0,0 +1,11 @@
+import {Octokit} from '@octokit/rest'
+import * as github from '@actions/github'
+
+export async function getPRAuthor(octokit: Octokit): Promise<string> {
+  const prInfo = await octokit.pulls.get({
```

Comment:
I'm not a huge fan of adding another API call but I guess if this is infrequently used it shouldn't matter as much